Domain License/Domain.License/Extensions/InsertLicenseCommandExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Linq;
+using Diagnosea.Submarine.Domain.License.Commands.InsertLicense;
+using Diagnosea.Submarine.Domain.License.Entities;
+
+namespace Diagnosea.Submarine.Domain.License.Extensions
+{
+    public static class InsertLicenseCommandExtensions
+    {
+        public static LicenseEntity ToEntity(this InsertLicenseCommand command)
+        {
+            return new LicenseEntity
+            {
+                Id = command.Id,
+                UserId = command.UserId,
+                Products = command.Products.Select(x => x.ToEntity()).ToList()
```

Comment:
What if this fails? What if there are no products and we're just creating a blank License?

Reply:
Given the list is always initialized, which there is no reason for it not to be, it will just never perform the `Select`.